pkg/controller/redirect/index_test.go

```diff
@@ -14,41 +14,77 @@
 
 package redirect
 
-import "testing"
+import (
+	"net/url"
+	"testing"
+)
 
-func TestBuildURL(t *testing.T) {
+func TestBuildEnsURL(t *testing.T) {
 	t.Parallel()
 
 	cases := []struct {
 		name   string
 		path   string
+		query  url.Values
 		region string
 		exp    string
 	}{
 		{
 			name:   "leading_slash_region",
 			path:   "v",
+			query:  url.Values{},
```

Comment:
In the future you could save yourself some typing by not including this line in your table since this is the default value.

Reply:
Ah, good point.
Done.